bundles/core/src/test/resources/form/panel/exporter-rules-panel.json

```diff
@@ -1,46 +1,58 @@
 {
-  "id": "panel-81f441fcab",
+  "id":"panel-81f441fcab",
+  "description":"dependent names description",
+  "tooltip":"dependent names tooltip",
+  "type":"object",
+  "fieldType":"panel",
+  "name":"abc",
+  "visible":true,
+  "enabled":true,
   "items": [
     {
-      "id": "textinput-bec585e910",
-      "description": "dummy",
-      "required": false,
-      "name": "abc",
-      "visible": false,
-      "enabled": true,
-      "readOnly": false,
-      "fieldType": "text-input",
-      "label": {
-        "visible": true,
-        "richText": false,
-        "value": "def"
+      "id":"textinput-bec585e910",
+      "description":"dummy",
+      "type":"string",
+      "fieldType":"text-input",
+      "required":false,
+      "name":"abc",
+      "visible":false,
+      "enabled":true,
+      "readOnly":false,
+      "label":{
+        "visible":true,
+        "richText":false,
+        "value":"def"
       },
-      "type": "string",
-      ":type": "core/fd/components/form/textinput/v1/textinput"
+      "properties":{
+        "fd:dor":{
+          "dorExclusion":false,
+          "dorColspan":null
```

Comment:
Why are we sending this when it is null. We should only include in json if it is not null

Reply:
By default in v1, dorExclusion and dorColspan contains no value. It gets value only if we explicitly set from authoring. So I guess this shouldn't be part of json by default.

Reply:
Irrespective of author/publish view, non-nullable values should only be part of json.